### DISCUSSION TITLE
Attention All Maintainers (Action Needed!)

### DISCUSSION THREAD

**Discussion by sgoudham** [catppuccin/catppuccin#2316] [Announcements]
> [!IMPORTANT]
> If you would like to continue as a maintainer/collaborator and are happy with your list of repositories, **do not reply** to this discussion. Feel free to unsubscribe from this discussion to avoid getting notifications for each reply.

Hey :wave:  @catppuccin/maintainers!

Catppuccin has grown _a lot_ and probably isn't stopping anytime soon either :sweat_smile:, we recognise that our maintainers for ports _will_ come and go. Of course, this is perfectly fine, but we want to explicitly track this going forward in the [ports.yml](https://github.com/catppuccin/catppuccin/blob/main/resources/ports.yml) file. A draft PR, #2315, goes into further detail about how we wish to track this. This gives maintainers (or future maintainers) the ability to raise pull requests in case they want start or stop maintaining port(s)

I've created a table, shown down below, which lists the names of collaborators against the list of repositories that they have `maintain` and/or `write` permissions on. **Please read the following headings and follow the instructions relevant to you.**

## If you are a current maintainer (`maintain` access)
1. Search for your GitHub username in the table below.
2. Confirm that you are happy with the list of repositories against your name.
3. **If you no longer want to be a maintainer**, please reply to this discussion listing what repositories you no longer want to maintain (or make it clear that you don't want to maintain any repositories.)

## If you are a collaborator (just `write` access)
1. Search for your GitHub username in the table below.
2. Confirm that you still intend to contribute to the repository.
3. **If you no longer want `write` permissions on a repository**, please reply to this discussion asking to be removed.

I will close this discussion as resolved once #2315 and https://github.com/catppuccin/userstyles/pull/706 are merged with all of the maintainers explicitly tracked.

<hr>

| Name | `Maintain` Access | `Write` Access |
| ---------- | ----------------- | -------------- |
| 112cxyz | [enmity](https://github.com/catppuccin/enmity) |  |
| 21st-centuryman | [sioyek](https://github.com/catppuccin/sioyek) |  |
| 89iuv | [tmux](https://github.com/catppuccin/tmux) | [nvim](https://github.com/catppuccin/nvim) |
| AdalZanabria | [mc](https://github.com/catppuccin/mc) |  |
| AlphaTechnolog | [rofi](https://github.com/catppuccin/rofi) |  |
| AngelBerihuete | [rstudio](https://github.com/catppuccin/rstudio) |  |
| Anomalocaridid | [delta](https://github.com/catppuccin/delta) |  |
| AnubisNekhet | [userstyles](https://github.com/catppuccin/userstyles) |  |
| Aurn1ox | [minecraft](https://github.com/catppuccin/minecraft) |  |
| BlankParticle | [userstyles](https://github.com/catppuccin/userstyles) |  |
| BlueFalconHD | [pantone](https://github.com/catppuccin/pantone), [anki](https://github.com/catppuccin/anki), [bento](https://github.com/catppuccin/bento), [gitui](https://github.com/catppuccin/gitui) |  |
| Brianalmeida | [rio](https://github.com/catppuccin/rio) |  |
| BrunDerSchwarzmagier | [sublime-text](https://github.com/catppuccin/sublime-text) |  |
| CallMeEchoCodes | [tty](https://github.com/catppuccin/tty), [minecraft](https://github.com/catppuccin/minecraft) |  |
| Daeraxa | [halloy](https://github.com/catppuccin/halloy) |  |
| Dandraghas | [userstyles](https://github.com/catppuccin/userstyles) |  |
| Delta456 | [v](https://github.com/catppuccin/v) |  |
| DonutDev | [lxqt](https://github.com/catppuccin/lxqt) |  |
| DorielRivalet | [imhex](https://github.com/catppuccin/imhex) |  |
| Dukeatron | [glamour](https://github.com/catppuccin/glamour) |  |
| @EdenEast |  | [nvim](https://github.com/catppuccin/nvim) |
| @Fadyio |  | [nvim](https://github.com/catppuccin/nvim) |
| FrancoRav | [userstyles](https://github.com/catppuccin/userstyles) |  |
| Gandalf-the-Blue | [userstyles](https://github.com/catppuccin/userstyles) |  |
| GenShibe | [userstyles](https://github.com/catppuccin/userstyles) |  |
| GitMuslim | [userstyles](https://github.com/catppuccin/userstyles) |  |
| Guaxinim5573 | [userstyles](https://github.com/catppuccin/userstyles) |  |
| InvitedToHell | [notepad-plus-plus](https://github.com/catppuccin/notepad-plus-plus) |  |
| JK-Flip-Flop96 | [powershell](https://github.com/catppuccin/powershell) |  |
| James-McK | [musicbee](https://github.com/catppuccin/musicbee) |  |
| @Jint-lzxy |  | [nvim](https://github.com/catppuccin/nvim) |
| JoshPaulie | [starship](https://github.com/catppuccin/starship) |  |
| Kettukaa | [stable-diffusion-webui](https://github.com/catppuccin/stable-diffusion-webui) |  |
| LudoPinelli | [windows-terminal](https://github.com/catppuccin/windows-terminal) |  |
| M3nny | [vim](https://github.com/catppuccin/vim) |  |
| Mqisty | [prismlauncher](https://github.com/catppuccin/prismlauncher) |  |
| NamesCode | [emacs](https://github.com/catppuccin/emacs) |  |
| NickSquiggles | [scrivener](https://github.com/catppuccin/scrivener) |  |
| PanAeon | [drracket](https://github.com/catppuccin/drracket) |  |
| Prayag2 | [kde](https://github.com/catppuccin/kde) |  |
| Quinxxxx | [rboard](https://github.com/catppuccin/rboard), [vendetta](https://github.com/catppuccin/vendetta) |  |
| RoootTheFox | [userstyles](https://github.com/catppuccin/userstyles) |  |
| Sekki21956 | [neomutt](https://github.com/catppuccin/neomutt), [spotify-tui](https://github.com/catppuccin/spotify-tui), [userstyles](https://github.com/catppuccin/userstyles) |  |
| ShyyLexi | [firefox](https://github.com/catppuccin/firefox) |  |
| Sourcastic | [kde](https://github.com/catppuccin/kde), [ksyntaxhighlighting](https://github.com/catppuccin/ksyntaxhighlighting) |  |
| Stonks3141 | [egui](https://github.com/catppuccin/egui), [aerc](https://github.com/catppuccin/aerc) |  |
| Syndrizzle | [gtk](https://github.com/catppuccin/gtk), [qt5ct](https://github.com/catppuccin/qt5ct), [flow-launcher](https://github.com/catppuccin/flow-launcher) |  |
| TarunDaCoder | [contour](https://github.com/catppuccin/contour) |  |
| TheExistingOne | [konsole](https://github.com/catppuccin/konsole) |  |
| TheMemeSniper | [userstyles](https://github.com/catppuccin/userstyles) |  |
| Thomas-Philippot | [userstyles](https://github.com/catppuccin/userstyles) |  |
| Tnixc | [blockbench](https://github.com/catppuccin/blockbench), [userstyles](https://github.com/catppuccin/userstyles) |  |
| ToxicAven | [discord](https://github.com/catppuccin/discord) |  |
| ValentinTT | [ida-debugger](https://github.com/catppuccin/ida-debugger) |  |
| VictorTennekes | [alacritty](https://github.com/catppuccin/alacritty), [iterm](https://github.com/catppuccin/iterm), [monkeytype](https://github.com/catppuccin/monkeytype) |  |
| Waxxx333 | [geany](https://github.com/catppuccin/geany) |  |
| Xurdejl | [infinity](https://github.com/catppuccin/infinity), [obs](https://github.com/catppuccin/obs), [qbittorrent](https://github.com/catppuccin/qbittorrent) |  |
| Zazucki | [speedcrunch](https://github.com/catppuccin/speedcrunch) |  |
| abhishekmj303 | [github-readme-streak-stats](https://github.com/catppuccin/github-readme-streak-stats) |  |
| @adityastomar67 |  | [nvim](https://github.com/catppuccin/nvim) |
| alythical | [userstyles](https://github.com/catppuccin/userstyles) |  |
| asrvd | [hyper](https://github.com/catppuccin/hyper), [sumatra-pdf](https://github.com/catppuccin/sumatra-pdf) |  |
| aszecsei | [hexchat](https://github.com/catppuccin/hexchat) |  |
| ~~ayamir~~ | [fcitx5](https://github.com/catppuccin/fcitx5) |  |
| bagelwaffle | [ghostwriter](https://github.com/catppuccin/ghostwriter) |  |
| bartlibert | [userstyles](https://github.com/catppuccin/userstyles) |  |
| @bashbunni |  | [nvim](https://github.com/catppuccin/nvim) |
| brambozz | [matplotlib](https://github.com/catppuccin/matplotlib) |  |
| caarlos0 | [go](https://github.com/catppuccin/go) |  |
| catb00mer | [misskey](https://github.com/catppuccin/misskey) |  |
| cattokomo | [diohub](https://github.com/catppuccin/diohub), [lua](https://github.com/catppuccin/lua) |  |
| coldenate | [remnote](https://github.com/catppuccin/remnote), [userstyles](https://github.com/catppuccin/userstyles), [pomotroid](https://github.com/catppuccin/pomotroid) |  |
| coopw1 | [userstyles](https://github.com/catppuccin/userstyles) |  |
| daveyholler | [nova](https://github.com/catppuccin/nova) |  |
| ~~davidbgonz~~ | [spicetify](https://github.com/catppuccin/spicetify) |  |
| daylinmorgan | [nim](https://github.com/catppuccin/nim) |  |
| denis-g | [forklift-4](https://github.com/catppuccin/forklift-4) |  |
| didair | [ui](https://github.com/catppuccin/ui) |  |
| djflan | [visual-studio](https://github.com/catppuccin/visual-studio) |  |
| elkrien | [vimium](https://github.com/catppuccin/vimium), [grub](https://github.com/catppuccin/grub), [chrome](https://github.com/catppuccin/chrome), [plank](https://github.com/catppuccin/plank), [mailspring](https://github.com/catppuccin/mailspring), [micro](https://github.com/catppuccin/micro), [cursors](https://github.com/catppuccin/cursors), [papirus-folders](https://github.com/catppuccin/papirus-folders), [spotify-player](https://github.com/catppuccin/spotify-player), [thunderbird](https://github.com/catppuccin/thunderbird), [Kvantum](https://github.com/catppuccin/Kvantum), [tilix](https://github.com/catppuccin/tilix), [blackbox](https://github.com/catppuccin/blackbox), [pyradio](https://github.com/catppuccin/pyradio) |  |
| @emxxjnm |  | [nvim](https://github.com/catppuccin/nvim) |
| fapfaff | [home-assistant](https://github.com/catppuccin/home-assistant), [refind](https://github.com/catppuccin/refind) |  |
| fathulfahmy | [glazewm](https://github.com/catppuccin/glazewm) |  |
| @fioncat |  | [nvim](https://github.com/catppuccin/nvim) |
| gabrielmagno | [skim](https://github.com/catppuccin/skim), [jupyterlab](https://github.com/catppuccin/jupyterlab) |  |
| ghishadow | [lapce](https://github.com/catppuccin/lapce) |  |
| griimick | [logseq](https://github.com/catppuccin/logseq), [codemirror](https://github.com/catppuccin/codemirror) |  |
| ~~hyperreal64~~ | [amfora](https://github.com/catppuccin/amfora), [zsh-syntax-highlighting](https://github.com/catppuccin/zsh-syntax-highlighting), [thelounge](https://github.com/catppuccin/thelounge) |  |
| ignamartinoli | [newsboat](https://github.com/catppuccin/newsboat) |  |
| iruzo | [crt](https://github.com/catppuccin/crt), [bemenu](https://github.com/catppuccin/bemenu), [simplex](https://github.com/catppuccin/simplex), [tofi](https://github.com/catppuccin/tofi) |  |
| isabelroses | [sddm](https://github.com/catppuccin/sddm), [freshrss](https://github.com/catppuccin/freshrss), [userstyles](https://github.com/catppuccin/userstyles), [swaync](https://github.com/catppuccin/swaync), [opera-gx](https://github.com/catppuccin/opera-gx) |  |
| itzTheMeow | [userstyles](https://github.com/catppuccin/userstyles) |  |
| jayylmao | [dopamine](https://github.com/catppuccin/dopamine), [nilesoft-shell](https://github.com/catppuccin/nilesoft-shell) |  |
| jn-sena | [userstyles](https://github.com/catppuccin/userstyles) |  |
| justTOBBI | [polybar](https://github.com/catppuccin/polybar), [remnote](https://github.com/catppuccin/remnote), [nighttab](https://github.com/catppuccin/nighttab), [xed](https://github.com/catppuccin/xed), [userstyles](https://github.com/catppuccin/userstyles) |  |
| justleoo | [tym](https://github.com/catppuccin/tym) |  |
| @kerichdev | [youtubemusic](https://github.com/catppuccin/youtubemusic) |  |
| kkrypt0nn | [binary-ninja](https://github.com/catppuccin/binary-ninja), [cutter](https://github.com/catppuccin/cutter), [fleet](https://github.com/catppuccin/fleet), [powerpoint-slides](https://github.com/catppuccin/powerpoint-slides), [linear](https://github.com/catppuccin/linear) |  |
| lemon-sh | [mirc](https://github.com/catppuccin/mirc) |  |
| lewisakura | [theme.park](https://github.com/catppuccin/theme.park) |  |
| lighttigerXIV | [ulauncher](https://github.com/catppuccin/ulauncher), [ueli](https://github.com/catppuccin/ueli), [fitbit](https://github.com/catppuccin/fitbit) |  |
| lmnystpd | [gitui](https://github.com/catppuccin/gitui) |  |
| lokesh-krishna | [zathura](https://github.com/catppuccin/zathura), [btop](https://github.com/catppuccin/btop), [mako](https://github.com/catppuccin/mako) |  |
| lucasmelin | [userstyles](https://github.com/catppuccin/userstyles) |  |
| @luzikii |  | [vendetta](https://github.com/catppuccin/vendetta) |
| mainrs | [zellij](https://github.com/catppuccin/zellij) |  |
| mbeckrich | [obsidian](https://github.com/catppuccin/obsidian) |  |
| mekb-turtle | [limine](https://github.com/catppuccin/limine) |  |
| meme8383 | [spyder](https://github.com/catppuccin/spyder) |  |
| mrdoge515 | [FluentTerminal](https://github.com/catppuccin/FluentTerminal) |  |
| mrtnvgr | [nvim](https://github.com/catppuccin/nvim) |  |
| mvilk | [minecraft](https://github.com/catppuccin/minecraft) |  |
| ndsboy | [plymouth](https://github.com/catppuccin/plymouth), [solvespace](https://github.com/catppuccin/solvespace), [heroic](https://github.com/catppuccin/heroic), [userstyles](https://github.com/catppuccin/userstyles) |  |
| nekowinston | [gnome-terminal](https://github.com/catppuccin/gnome-terminal), [vscode](https://github.com/catppuccin/vscode), [wezterm](https://github.com/catppuccin/wezterm), [tailwindcss](https://github.com/catppuccin/tailwindcss), [highlightjs](https://github.com/catppuccin/highlightjs), [alfred](https://github.com/catppuccin/alfred), [qutebrowser](https://github.com/catppuccin/qutebrowser), [noir](https://github.com/catppuccin/noir), [bat](https://github.com/catppuccin/bat), [homebrew-tap](https://github.com/catppuccin/homebrew-tap), [k9s](https://github.com/catppuccin/k9s), [warp](https://github.com/catppuccin/warp), [Terminal.app](https://github.com/catppuccin/Terminal.app), [mattermost](https://github.com/catppuccin/mattermost), [bottom](https://github.com/catppuccin/bottom), [sc-im](https://github.com/catppuccin/sc-im), [cider](https://github.com/catppuccin/cider), [gitea](https://github.com/catppuccin/gitea), [zsh-fsh](https://github.com/catppuccin/zsh-fsh) |  |
| night-lake | [userstyles](https://github.com/catppuccin/userstyles) |  |
| @npv12 |  | [gtk](https://github.com/catppuccin/gtk) |
| nullchilly | [nvim](https://github.com/catppuccin/nvim) |  |
| nullishamy | [thunderbird](https://github.com/catppuccin/thunderbird), [lazygit](https://github.com/catppuccin/lazygit), [serenity-os](https://github.com/catppuccin/serenity-os) |  |
| nyxkrage | [emacs](https://github.com/catppuccin/emacs) |  |
| ohitstom | [spicetify](https://github.com/catppuccin/spicetify) |  |
| ohxxm | [conky](https://github.com/catppuccin/conky), [helix](https://github.com/catppuccin/helix), [dmenu](https://github.com/catppuccin/dmenu) |  |
| opakholis | [insomnia](https://github.com/catppuccin/insomnia) |  |
| ozwaldorf | [steam-deck](https://github.com/catppuccin/steam-deck) |  |
| prateektade | [lxterminal](https://github.com/catppuccin/lxterminal), [openbox](https://github.com/catppuccin/openbox), [qterminal](https://github.com/catppuccin/qterminal) |  |
| prazdevs | [vscode-icons](https://github.com/catppuccin/vscode-icons) |  |
| pspiagicw | [emacs](https://github.com/catppuccin/emacs) |  |
| quentinguidee | [jetbrains](https://github.com/catppuccin/jetbrains), [xcode](https://github.com/catppuccin/xcode), [unreal-engine](https://github.com/catppuccin/unreal-engine), [jetbrains-icons](https://github.com/catppuccin/jetbrains-icons) |  |
| r58Playz | [userstyles](https://github.com/catppuccin/userstyles) |  |
| rayhanadev | [raycast](https://github.com/catppuccin/raycast) |  |
| remiposo | [swaylock](https://github.com/catppuccin/swaylock) |  |
| reniivali | [bitburner](https://github.com/catppuccin/bitburner) |  |
| riivx | [vendetta](https://github.com/catppuccin/vendetta) |  |
| rithikasilva | [blink](https://github.com/catppuccin/blink) |  |
| rockquiet | [userstyles](https://github.com/catppuccin/userstyles) |  |
| rubyowo | [gtk](https://github.com/catppuccin/gtk), [discord](https://github.com/catppuccin/discord), [waybar](https://github.com/catppuccin/waybar), [hyprland](https://github.com/catppuccin/hyprland), [youtubemusic](https://github.com/catppuccin/youtubemusic), [i3](https://github.com/catppuccin/i3), [userstyles](https://github.com/catppuccin/userstyles) |  |
| ryanccn | [userstyles](https://github.com/catppuccin/userstyles) |  |
| sacerd-OS | [gedit](https://github.com/catppuccin/gedit) |  |
| sakkke | [zutty](https://github.com/catppuccin/zutty), [tabby](https://github.com/catppuccin/tabby), [github-readme-tech-stack](https://github.com/catppuccin/github-readme-tech-stack), [vivaldi](https://github.com/catppuccin/vivaldi) |  |
| ~~schoblaska~~ | [base16](https://github.com/catppuccin/base16) |  |
| skelebro1 | [userstyles](https://github.com/catppuccin/userstyles) |  |
| skinatro | [floris-board](https://github.com/catppuccin/floris-board) |  |
| @sm185592 |  | [powershell](https://github.com/catppuccin/powershell) |
| sofycodes | [aseprite](https://github.com/catppuccin/aseprite) |  |
| soradotwav | [steam-deck](https://github.com/catppuccin/steam-deck) |  |
| southqaw | [yakuake](https://github.com/catppuccin/yakuake) |  |
| soya-daizu | [userstyles](https://github.com/catppuccin/userstyles) |  |
| spaghettiosareyummy | [userstyles](https://github.com/catppuccin/userstyles) |  |
| tecandrew | [zed](https://github.com/catppuccin/zed) |  |
| tetov | [fuzzel](https://github.com/catppuccin/fuzzel) |  |
| thismoon | [userstyles](https://github.com/catppuccin/userstyles) |  |
| tiepp | [jetbrains](https://github.com/catppuccin/jetbrains) |  |
| trinkey | [userstyles](https://github.com/catppuccin/userstyles) |  |
| tsjazil | [st](https://github.com/catppuccin/st) |  |
| uncenter | [userstyles](https://github.com/catppuccin/userstyles), [yazi](https://github.com/catppuccin/yazi) |  |
| uncomfyhalomacro | [kakoune](https://github.com/catppuccin/kakoune) |  |
| useEffects | [flutter](https://github.com/catppuccin/flutter) |  |
| vdbe | [tmux](https://github.com/catppuccin/tmux) |  |
| @vinibispo |  | [nvim](https://github.com/catppuccin/nvim) |
| @vollowx |  | [nvim](https://github.com/catppuccin/nvim) |
| walshyb | [latex](https://github.com/catppuccin/latex) |  |
| watatomo | [discord](https://github.com/catppuccin/discord), [userstyles](https://github.com/catppuccin/userstyles) |  |
| xhayper | [moon-animator-2](https://github.com/catppuccin/moon-animator-2) |  |
| @andreasgrafen | [revolt](https://github.com/catppuccin/revolt) |  |
| @backwardspy | [fish](https://github.com/catppuccin/fish), [rust](https://github.com/catppuccin/rust), [python](https://github.com/catppuccin/python), [godot](https://github.com/catppuccin/godot), [qtcreator](https://github.com/catppuccin/qtcreator), [dwarf-fortress](https://github.com/catppuccin/dwarf-fortress) |  |
| @ghostx31 | [joplin](https://github.com/catppuccin/joplin), [foliate](https://github.com/catppuccin/foliate), [element](https://github.com/catppuccin/element) |  |
| @isabelincorp | [oldtwitter](https://github.com/catppuccin/oldtwitter) |  |
| @sgoudham | [mdBook](https://github.com/catppuccin/mdBook), [sharex](https://github.com/catppuccin/sharex), [slack](https://github.com/catppuccin/slack), [java](https://github.com/catppuccin/java), [windows-files](https://github.com/catppuccin/windows-files), [glamour](https://github.com/catppuccin/glamour), [jetbrains](https://github.com/catppuccin/jetbrains) |  |
| @taka0o | [regolith](https://github.com/catppuccin/regolith) |  |
| [Core](https://github.com/orgs/catppuccin/teams/core)/[Staff](https://github.com/orgs/catppuccin/teams/staff) | [homebrew-tap](https://github.com/catppuccin/homebrew-tap), [.github](https://github.com/catppuccin/.github), [catppuccin](https://github.com/catppuccin/catppuccin), [template](https://github.com/catppuccin/template), [xresources](https://github.com/catppuccin/xresources), [kitty](https://github.com/catppuccin/kitty), [termux](https://github.com/catppuccin/termux), [urxvt](https://github.com/catppuccin/urxvt), [fish](https://github.com/catppuccin/fish), [palette](https://github.com/catppuccin/palette), [terminator](https://github.com/catppuccin/terminator), [cava](https://github.com/catppuccin/cava), [revolt](https://github.com/catppuccin/revolt), [telegram](https://github.com/catppuccin/telegram), [foot](https://github.com/catppuccin/foot), [website](https://github.com/catppuccin/website), [whoogle](https://github.com/catppuccin/whoogle), [fzf](https://github.com/catppuccin/fzf), [dark-reader](https://github.com/catppuccin/dark-reader), [dunst](https://github.com/catppuccin/dunst), [xfce4-terminal](https://github.com/catppuccin/xfce4-terminal), [github-readme-stats](https://github.com/catppuccin/github-readme-stats), [toolbox](https://github.com/catppuccin/toolbox), [windows-files](https://github.com/catppuccin/windows-files), [joplin](https://github.com/catppuccin/joplin), [mdBook](https://github.com/catppuccin/mdBook), [sharex](https://github.com/catppuccin/sharex), [cli](https://github.com/catppuccin/cli), [community](https://github.com/catppuccin/community), [element](https://github.com/catppuccin/element), [regolith](https://github.com/catppuccin/regolith), [foliate](https://github.com/catppuccin/foliate), [slack](https://github.com/catppuccin/slack), [rust](https://github.com/catppuccin/rust), [python](https://github.com/catppuccin/python), [dwarf-fortress](https://github.com/catppuccin/dwarf-fortress), [java](https://github.com/catppuccin/java), [ultrakill](https://github.com/catppuccin/ultrakill), [qtcreator](https://github.com/catppuccin/qtcreator), [replit](https://github.com/catppuccin/replit), [oldtwitter](https://github.com/catppuccin/oldtwitter), [godot](https://github.com/catppuccin/godot) |  |

**Comment by hyperreal64** [comment]
I'm currently the maintainer for amfora, zsh-syntax-highlighting, and The Lounge. I no longer wish to maintain any repositories at this time.

**Reply by sgoudham** [reply]
Thanks for letting me know, I have removed your permissions as requested and will remove you from the maintainers team.
It goes without saying but thank you for your efforts in creating those ports and, of course, you're welcome back anytime!

**Comment by jayylmao** [comment]
currently a maintainer for dopamine and nilesoft-shell. i want to maintain these.

**Comment by sofycat** [comment]
currently a maintainer for aseprite, I want to maintain these.

**Comment by bagelwaffle** [comment]
May I have maintainer access to the kakoune port?

**Reply by backwardspy** [reply]
granted!

**Comment by CallMeEchoCodes** [comment]
currently maintaining TTY. it is complete and shouldnt need any updates but i still want to maintain it just in case.

**Comment by Delta456** [comment]
I am a maintainer for v and I still want to maintain it.

**Comment by asrvd** [comment]
btw don't know if anyone saw that but it's written that you only need to reply if you no longer wanna maintain your repos -

**If you would like to continue as a maintainer/collaborator and are happy with your list of repositories, do not reply to this discussion.**

**Comment by davidbgonz** [comment]
I no longer wish to be a maintainer for the spicetify port. Thanks for taking the time to manage all of this.

**Reply by backwardspy** [reply]
thanks for all your work on it!

**Comment by ayamir** [comment]
I'm currently the maintainer for `fcitx5`. I no longer wish to maintain it b/c I don’t use `fcitx5` often anymore.

**Reply by sgoudham** [reply]
Thanks for letting me know! I've removed your maintainer permissions.
Thanks again for all your work!

**Comment by schoblaska** [comment]
I'm currently the maintainer for base16. I'm not actively maintaining it, so feel free to remove my maintainer permissions. Thanks for cleaning up!

**Comment by ghost** [comment]
I'm willing to continue maintaining gedit.

**Comment by rogeruiz** [comment]
I am maintainer of the Tmux project and would like to continue be an active maintainer. Thank you.

**Comment by rubyowo** [comment]
I do not maintain the YouTube music port any more.

**Comment by bartlibert** [comment]
Unfortunately, I don't have the free time to maintain startpage anymore, I'm sorry.

**Reply by isabelroses** [reply]
Thanks for your work, I've removed you as a maintainer from the userstyle.

**Comment by justTOBBI** [comment]
Hey, I'd like to sign out of remnote (give @coldenate maintainer), polybar, xed and nighttab :)  Thanks for the great interactions, but my focus has shifted  a bit away from catppuccin :))

**Reply by backwardspy** [reply]
thanks for all your hard work!

**Reply by isabelroses** [reply]
Just to confim @justTOBBI, you are still willing to maintain the codeberg and hoppscotch userstyles?

**Reply by justTOBBI** [reply]
@isabelroses I'd be down to maintain them, until someone else steps in :)

**Reply by isabelroses** [reply]
I can take over the codeberg userstyle, but I wouldn't be the best for hopscotch.

**Comment by vinibispo** [comment]
Hey @sgoudham,  I still want to continue working on the neovim port, so I still wish the write access

**Comment by jn-sena** [comment]
Hi. I'm sorry but I no longer want to maintain any ports here. Thank you so much, I had a good time!

**Reply by backwardspy** [reply]
thank you for your contributions!

**Reply by isabelroses** [reply]
Thanks for your work, just to confirm you are happy with, [moving you to past maintainers on the userstyles repo](https://github.com/catppuccin/userstyles/pull/704).

**Comment by ohxxm** [comment]
Sorry all but I don't have the time anymore to maintain any ports on Catppuccin. It was a great time while it lasted!

**Comment by AngelBerihuete** [comment]
I am maintainer of the Rstudio project and would like to continue be an active maintainer. Thank you.

**Comment by VictorTennekes** [comment]
Happy to have been here for the start of Catpuccin but currently do not have any capacity to maintain the ports I did. Will keep following to see what other amazing things Catpuccin achieves in the future.

**Reply by nekowinston** [reply]
@sgoudham happy to pick up Alacritty and iTerm since I did the past updates for those.

**Comment by AnubisNekhet** [comment]
@sgoudham Happy to pick up Anki since it hasn't been updated in 8 months.

**Comment by sgoudham** [comment]
Thanks to all those who commented in the discussion. I've now closed #2315. 

I think the accuracy of the `current-maintainers` key is pretty good, but `past-maintainers` was quite difficult to track down for all the ports :sweat_smile: More than happy to accept PRs from current/previous maintainers if they've not properly been attributed. 

Again, thanks everyone for continuing to care about Catppuccin. I hope this will make it easier for people to express their wish to join or leave the organisation!
